### PR TITLE
Fix for the no_network unit test

### PR DESCRIPTION
**Pull request by heidijinxujia**
Fix for the test testGetAuthTokenInBackgroundVerifyErrorNoNetworkConnection()

### CODE REVIEW

File: adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java [hunk 1]
```diff
@@ -673,14 +673,16 @@ public void testGetAuthTokenInBackgroundVerifyErrorMessageNotSupported() throws
...
+    public void testGetAuthTokenInBackgroundVerifyErrorNoNetworkConnection() throws NameNotFoundException,
+    OperationCanceledException, IOException, AuthenticatorException {
         final String acctName = "testAcct123";
         final AuthenticationRequest authRequest = createAuthenticationRequest("https://login.windows.net/omercantest", "resource", "client",
                 "redirect", acctName.toLowerCase(Locale.US), PromptBehavior.Auto, "", UUID.randomUUID(), false);
         final FileMockContext context = new FileMockContext(getContext());
         context.setConnectionAvaliable(false);
         final AccountManager mockedAccountManager = getMockedAccountManager(AuthenticationConstants.Broker.BROKER_ACCOUNT_TYPE,
                 AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME);
+        setMockProxyForErrorCheck(mockedAccountManager, acctName, AccountManager.ERROR_CODE_NETWORK_ERROR, ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE.getDescription());
```
Comment (addev-ashish) [inline adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java]: can you also add a e.getMessage()!= null in the place where the crash was as well.

Comment (heidijinxujia) [inline adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java]: Thank you. Addressed in the latest commit.

File: adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java [hunk 2]
```diff
@@ -338,7 +338,7 @@ private Bundle getAuthTokenFromAccountManager(final AuthenticationRequest reques
                 Logger.e(TAG, AUTHENTICATOR_CANCELS_REQUEST, "", ADALError.AUTH_FAILED_CANCELLED, e);
             } catch (AuthenticatorException e) {
                 Logger.e(TAG, AUTHENTICATOR_CANCELS_REQUEST, "", ADALError.BROKER_AUTHENTICATOR_NOT_RESPONDING);
-                if (e.getMessage().contains(ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE.getDescription())) {
+                if (e.getMessage()!= null && e.getMessage().contains(ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE.getDescription())) {
```
Comment (addev-ashish) [inline adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java]: >()! [](start = 32, length = 3)

there should be a space between () and !. One of the static analysis tools should be complaining. is it not?

**Comment by addev-ashish** [comment]
Fix the minor comment then :shipit:

**Comment by heidijinxujia** [comment]
Merge after Travis pass

**Comment by heidijinxujia** [comment]
Squash the commit and merge.

**Pull request merged**